packages/contracts-bedrock/src/constants.ts

```diff
@@ -19,6 +19,7 @@ export const predeploys = {
   LegacyERC20ETH: '0xDeadDeAddeAddEAddeadDEaDDEAdDeaDDeAD0000',
   WETH9: '0x4200000000000000000000000000000000000006',
   GovernanceToken: '0x4200000000000000000000000000000000000042',
+  ProxyAdmin: '0x4200000000000000000000000000000000000016',
```

Comment:
Comment only: it sucks that this has to be written 3 times. I have not solution for it though.

Reply:
Yeah... solidity, go and js